package.json

```
@@ -19,6 +19,7 @@
     "@babel/preset-react": "^7.0.0",
     "@babel/runtime": "^7.0.0",
     "@sentry/browser": "^5.0.6",
```

```suggestion
    "@sentry/browser": "^5.0.8",
```

Let's also bump browser, otherwise LGTM 👍